src/components/rails/explorer-rail/pmBlocksLoop/pmBlocksLoop.vue

```diff
@@ -28,8 +28,11 @@ export default {
           :key="object.id"
           @click.stop="onClickHandler($event, object)"
         >
-          <img class="node-types__item__icon" :src="object.icon" :alt="$t(object.label)">
-          <span>{{ $t(object.label) }}</span>
+          <div class="d-flex">
+            <i v-if="!svgIcon" class="node-types__item__icon" :class="object.customIcon"/>
```

Comment:
@mcraeteisha can we update this v-if conditional to reference the object.svgIcon so it would be like 
`<i v-if="!object.svgIcon" ....`

Reply:
Done! Thanks for catching.